lib/res/default_config.json

```diff
@@ -20,7 +20,7 @@
         "enabled": false,
         "server": "foobar.example",
         "port": 1883,
-        "clientId": "",
+        "clientId": "valetudo",
```

Comment:
Note that at least mosquitto (and maybe all mqtt brokers) will disconnect an existing session if a new one with the same clientId appears. That could be problematic in multi-robot setups.

Therefore, this should probably become a prefix/suffix to the identifier to avoid these situations

Reply:
That's a good idea. I'll set it to null by default and, if it's null, generate a random string.  To be fair though the client ID was static earlier too. It was using `identifier` instead of `clientID`, `clientId` was already there but it wasn't being used.

I decided to use `identifier` for the robot topic name instead.